redis/client.py

```diff
@@ -2789,6 +2790,10 @@ def zadd(self, name, mapping, nx=False, xx=False, ch=False, incr=False):
         if incr:
             pieces.append(b'INCR')
             options['as_score'] = True
+        if gt:
```

Comment:
Redis docs suggest these options are mutually exclusive. Should this be an if/elif?

Reply:
Fixed.